tests/e2e/CODE_STYLE.md

```diff
@@ -33,8 +33,8 @@ Automated lint checking and code format performs with ESLint and Prettier tools
 pre-commit hook.
 Full set of rules can be found:
 
-- [.eslintrc](.eslintrc.js)
-- [.prettierrc](.prettierrc.json)
+-   [.eslintrc](.eslintrc.js)
```

Comment:
What is the purpose of changing formatting in this file?

Reply:
Just is the result applying `npm run prettier`.
Working on PR we need to do:
```
nmp run lint
npm run prettier
```
So I decided to left changes in PR
that's it

Reply:
you meant npm

okay then